src/clp_ffi_js/ir/StreamReader.cpp

```diff
@@ -124,7 +124,7 @@ EMSCRIPTEN_BINDINGS(ClpStreamReader) {
             .value("STRUCTURED", clp_ffi_js::ir::StreamType::Structured)
             .value("UNSTRUCTURED", clp_ffi_js::ir::StreamType::Unstructured);
     emscripten::register_type<clp_ffi_js::ir::DecodedResultsTsType>(
-            "Array<[string, number, number, number]>"
+            "Array<[string, bigint, number, number]>"
```

Comment:
Note the behaviour with `logEventNum` of type `size_t` is different: https://github.com/y-scope/clp-ffi-js/issues/33#issuecomment-2466953640

Reply:
Should we change (perhaps in another PR) the log event indices into `uint32_t`? I feel like having more than 4B log events in a file is unlikely.

Reply:
> Should we change (perhaps in another PR) the log event indices into `uint32_t`?

Let's first investigate the behaviour after setting `-sMEMORY64=1`. If the underlying type of `size_t` changes to 64 bit unsigned, I think it makes sense to always use `uint32_t`.